src/redux/apiConnection.js

```diff
@@ -0,0 +1,59 @@
+import axios from 'axios';
+import { createAsyncThunk } from '@reduxjs/toolkit';
+
+export const postApiData = axios.post('https://us-central1-bookstore-api-e63c8.cloudfunctions.net/bookstoreApi/apps/');
+
+export const currentApiKey = 'gwE0Bhw9RcasmYxC9anP';
+
+const createBookList = (book) => {
+  const bookList = [];
+  const key = Object.keys(book);
+  key.forEach((item) => {
+    // console.log(item);
```

Comment:
- [ ] Please kindly remove `commented code` from your `code base`, as it makes your `code base` dirty and less readable. To make sure that your codebase is always clean, always remember to remove commented-codes.